jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -575,11 +575,19 @@ public ContinuationScope virtualThreadContinuationScope() {
 	}
 
 	public void parkVirtualThread() {
-		VirtualThread.park();
+		if (Thread.currentThread() instanceof BaseVirtualThread bvt) {
```

Comment:
is BaseVirtualThread a new type?

Reply:
Yes, it was just introduced via https://github.com/ibmruntimes/openj9-openjdk-jdk/commit/b1ef35620fae0fec3ddafe49a23f3faebfaebcc3 which is still in `openj9-staging`.